scripts/quick_install.sh

```diff
@@ -1,32 +1,31 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
we can use /bin/sh as it is defined as an available path in posix

Reply:
Okay sure I will change that